MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -503,7 +543,8 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
 
     // Show the alert controller.
     MSLogDebug([MSDistribute logTag], @"Show update dialog.");
-    [alertController show];
+    [alertController replaceAlert:self.updateAlertController];
```

Comment:
This is to cover the case where: You have no network > start the app > the persisted mandatory update alert is displayed > then the network comes back > a new release is there > we need to replace the old alert with the new one.

That's it, we need to remember the old alert to be able to replace it later if there is need.